ambry-store/src/test/java/com.github.ambry.store/IndexSegmentTest.java

```diff
@@ -51,6 +51,7 @@
  */
 @RunWith(Parameterized.class)
 public class IndexSegmentTest {
+  private static final String BLOOM_FILE_NAME_SUFFIX = "bloom";
```

Comment:
no need to declare a new variable. The variable in `IndexSegment` is package private.

Reply:
true, fixed.